.chloggen/branch-issue-32716.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: 'receiver/mongodbreceiver'
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: ' Validated new versions 6.0 and 7.0 of Mongodb for Mongodb receiver and updated integration tests'
```

How does this sound?
```suggestion
note: 'Ensure support of 6.0 and 7.0 Mongodb versions with integration tests'
```

---

sounds concise and to the point, thank you!